platform-sdk/docs/proposals/TSS-Library/TSS-Library.md

```diff
@@ -890,7 +731,8 @@ After this proposal is accepted we will invite security team to define the neces
 ## Alternatives Considered
 
 - As mentioned before, Java Foreign Function Interface is still immature so it was not considered for this proposal.
-- Using JCA through Bouncy Castle implementation was analyzed. BC provides support for EC-Curves, but it does not support Pairings which are required for BLS signatures.
+- JRA was discarded given the possibility of eventually moving to JFF.
+- Using JSA through Bouncy Castle implementation was analyzed. BC provides support for EC-Curves, but it does not support Pairings which are required for BLS signatures.
```

Comment:
It may be worth noting briefly without going into too much detail somewhere which signature scheme we are using and why it works best for us ? e.g. BLS >> Schnorr >> ECDSA

Reply:
Thanks for the Note, @thomas-swirlds-labs; I'll find somewhere to add a note.
Note: It's BLS.